### PR TITLE
telemetry(amazonq): human in the loop

### PR DESCRIPTION
**Pull request by damntrecky**
## Problem
We have a new feature going out and want to log metrics associated to that feature.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: telemetry/definitions/commonDefinitions.json [hunk 1]
```diff
@@ -1311,6 +1311,153 @@
...
+            "type": "string",
+            "description": "Type of maven command",
+            "allowedValues": [
+                "mvnw.cmd",
+                "mvnw",
+                "mvn",
+                "IDEBundledMaven"
+            ]
+        },
+        {
```
Comment (damntrecky) [inline telemetry/definitions/commonDefinitions.json:1456]: This is the only new update, the rest of changes must be a spacing change?

File: telemetry/definitions/commonDefinitions.json [hunk 2]
```diff
@@ -3146,6 +3293,24 @@
                 }
             ]
         },
+        {
```
Comment (damntrecky) [inline telemetry/definitions/commonDefinitions.json:3296]: This is the new metric added.

File: telemetry/definitions/commonDefinitions.json [hunk 3]
```diff
@@ -1311,6 +1311,153 @@
...
+                "apiError",
+                "cancelButton"
+            ]
+        },
+        {
+            "name": "codeTransformApiNames",
+            "type": "string",
+            "description": "Names of allowed API calls",
+            "allowedValues": [
+                "ResumeTransformation",
```
Comment (damntrecky) [inline telemetry/definitions/commonDefinitions.json:1419]: `ResumeTransformation` value was new as well.

Comment (justinmk3) [reply telemetry/definitions/commonDefinitions.json:1419]: ? definitely not a spacing change. there are lots of duplicate metrics added in this PR

File: telemetry/definitions/commonDefinitions.json (hunk 1 above)
Comment (justinmk3) [reply telemetry/definitions/commonDefinitions.json:1456]: ? definitely not a spacing change. there are lots of duplicate metrics added in this PR

Comment (damntrecky) [reply telemetry/definitions/commonDefinitions.json]: I didn't catch this. My main branch fork had the duplicates on the remote. FIXED.

File: telemetry/definitions/commonDefinitions.json [hunk 4]
```diff
@@ -3146,6 +3152,24 @@
                 }
             ]
         },
+        {
+            "name": "codeTransform_humanInTheLoop",
+            "description": "An interactivity measured when a pause state is enabled that requires human interaction.",
```
Comment (justinmk3) [inline telemetry/definitions/commonDefinitions.json:3157]: what does this sentence mean?

File: telemetry/definitions/commonDefinitions.json [hunk 5]
```diff
@@ -3146,6 +3152,24 @@
                 }
             ]
         },
+        {
+            "name": "codeTransform_humanInTheLoop",
```
Comment (justinmk3) [inline telemetry/definitions/commonDefinitions.json:3156]: can this be a field on an existing metric? it's not clear from the description what its purpose is

File: telemetry/definitions/commonDefinitions.json (hunk 4 above)
Comment (damntrecky) [reply telemetry/definitions/commonDefinitions.json]: Updated `            "description": "The user interactivity when a transformation pause state is reached.",
`

**Review by justinmk3 — APPROVED** [review]

**Comment by justinmk3** [comment]
> "The user interactivity when a transformation pause state is reached.",

Almost all metrics represent human interactions. So this is hard to understand.

It still seems like this could be metadata on an existing metric, e.g. `codeTransform_jobStatusChanged` or some other metric that captures the "transform" flow.

But it looks like most of these `codeTransform_` metrics are capturing very, very narrow concepts.

**Pull request merged**